### PR TITLE
feat: #224 check for prod firebase url

### LINKED ISSUE #224: Handle env config for Firebase deployments
We currently use test or live config based on the Cosmos live URL (digital-paper-edit.tools.bbc.co.uk) - we should extend this for the Firebase-hosted deployment (digital-paper-edit-prod.firebaseapp.com) by changing the isProduction function)

### PR DESCRIPTION
**Pull request by aniablaziak**
**Is your Pull Request request related to another [issue](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**      

Closes #224 

**Describe what the PR does**    

Check for Firebase production URL to serve the right config 

**State whether the PR is ready for review or whether it needs extra work**    

Ready

**Additional context**

### CODE REVIEW

**Review by allishultes — APPROVED** [review]
Looks ace to me, Anna!

**Review by tamsingreen — APPROVED** [review]
LGTM!

**Comment by aniablaziak** [comment]
cheers! 🙂

File: src/Util/is-production/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import url from 'url';
 const isProduction = () => {
   const parsedUrl = url.parse(window.location.href, true);
 
-  return parsedUrl.host === 'digital-paper-edit.tools.bbc.co.uk';
+  return parsedUrl.host === 'digital-paper-edit.tools.bbc.co.uk' || parsedUrl.host === 'digital-paper-edit-prod.firebaseapp.com';
```
Comment (bevand10) [inline src/Util/is-production/index.js:6]: Here, window.location.host will provide hostname with zero dependency on a 3rd party url library.

**Pull request merged**